### DISCUSSION TITLE
Workflow overview/list has wrong order

### DISCUSSION THREAD

**Discussion by PetratDNA** [community/community#11545] [Actions]
See 
![image](https://user-images.githubusercontent.com/89917593/154099790-be64a8ac-9156-4ccd-8498-9af2e395ed92.png)
Instead of ignoring case, the GHA orders uppercase first and then lowercase

**Comment by ethomson** [comment]
Indeed we do - this matches the behavior of sorting elsewhere on the platform, for example in the repository browser.

<img width="899" alt="Screen Shot 2022-02-15 at 11 57 47" src="https://user-images.githubusercontent.com/1130014/154110637-5a53fa5b-9ead-406a-88c3-5fab261d3cdd.png">

We can look at giving you some better ways to sort and visualize these in the future.

**Reply by PetratDNA** [reply]
Thanks for reply. I see.
